usr/share/rear/layout/save/default/950_verify_disklayout_file.sh

```diff
@@ -0,0 +1,168 @@
+#
+# Verify that the entries in disklayout.conf match syntactically
+# what is specified in the section "Disk layout file syntax"
+# in doc/user-guide/06-layout-configuration.adoc
+# cf. https://github.com/rear/rear/issues/2006#issuecomment-460646685
+#
+# Verify that for each 'disk' the 'part' entries in disklayout.conf
+# specify consecutive partition device nodes for the disk
+# because otherwise "rear recover" would fail with 'parted' error
+#   Error: Partition doesn't exist
+# cf. https://github.com/rear/rear/issues/1681
+#
+
+LogPrint "Verifying that the entries in $DISKLAYOUT_FILE are correct ..."
+local keyword dummy junk
+
+Log "Verifying that the 'disk' entries in $DISKLAYOUT_FILE are correct"
+# The section "Disk layout file syntax" in doc/user-guide/06-layout-configuration.adoc reads (excerpt)
+#   disk <name> <size(B)> <partition label> 
```

Comment:
There is an empty space at the end of this line ;-). My OCD suffers :-)

Reply:
Such severe coding faults deserve a dedicated commit:
https://github.com/rear/rear/pull/2060/commits/561c7482063b9570231f18c44d33058c264dc205
;-)

@gozora 
thank you for having in a look!

I hopefully fixed some real errors in
https://github.com/rear/rear/pull/2060/commits/4db41676bb0d220d12b573b75d4e1f862a7a3b2c
but I did not yet test it...

Reply:
> Such severe coding faults deserve a dedicated commit:

Thank you!

I'll do couple of tests, with this PR later today or maybe during weekend ...

V.